From c142713076b2b924e8ba44295164a0abca1355eb Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 16 Jan 2023 21:51:02 +0200
Subject: [PATCH] [SW-99614] gaudi2: security: fix whitespace

No need to indent the array definition closing bracket.
No funcitonal changes - only white space.

Change-Id: I44abac3d7c9a89a5031b4086f5f9b2e612027f10
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/common/security.c b/arch/gaudi2/common/security.c
index 83b8697..eda7331 100644
--- a/arch/gaudi2/common/security.c
+++ b/arch/gaudi2/common/security.c
@@ -187,7 +187,7 @@
 	mmPSOC_SECURITY_PUBLIC_KEY_1_0,
 	mmPSOC_SECURITY_PUBLIC_KEY_2_0,
 	mmPSOC_SECURITY_PUBLIC_KEY_3_0
-	};
+};
 
 static void hl_secure_update_hit_reg(int rr_idx, int inst_dist,
 		struct hit_regs *rr_reg, bool write_only)
